2-collectors/scala-stream-collector/project/Dependencies.scala

```diff
@@ -33,7 +33,7 @@ object Dependencies {
     val awsSdk           = "1.6.10"
     val yodaTime         = "2.1"
     val yodaConvert      = "1.2"
-    val kafka            = "0.10.1.0"
+    val kafka            = "0.10.2.0"
```

Comment:
@BenFradet no not really. If you don't support this upgrade, you can't support security by passing the jaas conf as a property from producerProps